common/views/components/CompactCard/CompactCard.js

```diff
@@ -31,6 +31,7 @@ type Props = {|
   StatusIndicator: ?Element<typeof StatusIndicator>,
   ExtraInfo?: ?Node,
   xOfY: {| x: number, y: number |},
+  type?: ?string,
```

Comment:
Where possible, we've tried to make a type _either_ optional _or_ nullable, and try to avoid having both.